modules/tracktion_engine/utilities/tracktion_Engine.h

```diff
@@ -33,15 +45,15 @@ class Engine
    #endif
     static juce::Array<Engine*> getEngines();
 
-    TemporaryFileManager& getTemporaryFileManager() const;
-    AudioFileFormatManager& getAudioFileFormatManager() const;
-    PropertyStorage& getPropertyStorage() const;
-    UIBehaviour& getUIBehaviour() const;
-    EngineBehaviour& getEngineBehaviour() const;
-    DeviceManager& getDeviceManager() const;
-    MidiProgramManager& getMidiProgramManager() const;
-    ExternalControllerManager& getExternalControllerManager() const;
-    RenderManager& getRenderManager() const;
+    TemporaryFileManager& getTemporaryFileManager() const; ///< Get the shared temporary file manager allowing to handle default app and user temporary folders.
```

Comment:
Most of the comments in this section have grammatical mistakes that will jump out to any native English speakers as being wrong... 

I really hate to criticise something like that, and if it was just a comment on some internal code, I'd ignore it because it's clear what you meant.. but any new content that's intended for end-user-facing documentation really needs to be 100% correct.

Reply:
OK, will sure revisit all use of MIDI and propose fixes ASAP, for the comment backward style any remarks?

i.e.  do you like the backward comment style (the idea is to not disrupt too much the code length and let those not interested to read focus on the left-side.

For grammar, will also be keen on any remark to make it better too, thanks for that.